UmatiServerLib/OpcUaKeys.cpp

```diff
@@ -143,10 +143,14 @@ std::vector<UA_ByteString> OpcUaKeys::readDir(std::string dirname) {
 }
 
 void OpcUaKeys::GenerateKeys() {
-  UA_String subject[3] = {UA_STRING_STATIC("C=DE"), UA_STRING_STATIC("O=SampleOrganization"), UA_STRING_STATIC("CN=UmatiSampleServer@localhost")};
+  UA_String subject[4] = {
+    UA_STRING_STATIC("C=DE"),
+    UA_STRING_STATIC("O=SampleOrganization"),
+    UA_STRING_STATIC("CN=umatiSampleServer@localhost"),
+    UA_STRING_STATIC("L=Frankfurt/Main")};
 
-  UA_UInt32 lenSubject = 3;
-  UA_String subjectAltName[2] = {UA_STRING_STATIC("DNS:localhost"), UA_STRING_STATIC("URI:urn:UmatiSampleServer")};
+  UA_UInt32 lenSubject = 4;
+  UA_String subjectAltName[2] = {UA_STRING_STATIC("DNS:opcserver"), UA_STRING_STATIC("URI:urn:umatiSampleServer")};
```

Comment:
Why has `localhost` been changed? opcserver is not a valid DNS name, so this will never match the hostname of the OPC UA-Server.

Reply:
Thanks for the catch. This was a leftover from a test.